flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/SubtaskCheckpointCoordinatorImpl.java

```diff
@@ -384,4 +478,12 @@ private static OperatorSnapshotFutures checkpointStreamOperator(
 			throw ex;
 		}
 	}
+
+	private static class NotifiedCheckpointAbortedException extends Exception {
```

Comment:
Couldn't find any usages (remove?).

Reply:
good catch, this is something implemented in previous commits but already useless.